providers/flagd/src/test/java/dev/openfeature/contrib/providers/flagd/e2e/RunFlagdRpcCucumberTest.java

```diff
@@ -16,9 +16,10 @@
 @Order(value = Integer.MAX_VALUE)
 @Suite
 @IncludeEngines("cucumber")
-@SelectClasspathResource("features/evaluation.feature")
-@SelectClasspathResource("features/flagd-json-evaluator.feature")
-@SelectClasspathResource("features/flagd.feature")
+//@SelectClasspathResource("features/evaluation.feature")
+//@SelectClasspathResource("features/flagd-json-evaluator.feature")
+//@SelectClasspathResource("features/flagd.feature")
```

Comment:
Are these meant to be commented? We're losing a bunch of coverage here right? Maybe you just didn't mean to commit this and just did it to speed up your local testing.